test/parser/mixins.test.js

```diff
@@ -175,6 +175,34 @@ test('!important, no whitespace', (t) => {
   t.is(nodeToString(root), less);
 });
 
+test('!important, whitespace between', (t) => {
+  const less = `
+                    .foo()! important;
```

Comment:
I haven't been able to find documentation in or out of the spec that says this form is valid (with no leading space before the !) and postcss doesn't parse that with standard declarations. I know this was part of the last major version, but it was intentionally left out of the current major because there wasn't any docs I could find.

Reply:
Hmm, I'm getting back valid results from both `postcss`, the official `less` compiler, as well as browsers when omitting the leading space. I guess the closest to a spec on it would be https://www.w3.org/TR/css-syntax-3/#declaration-diagram but I agree that it's not superclear.